src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java

```diff
@@ -41,36 +35,18 @@ public void getFixedFieldsFromLeader_return200Status() {
     String url = getURI(FIXED_FIELDS_CODE_GROUPS_URL);
     Map<String, String> headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
 
-    given()
-      .param("leader", "01657nam a22002057i 4500")
-      .param("code", "000")
-      .param("headerTypeCode", "1")
-      .param("lang", "ita")
-      .headers(headers)
-      .when()
-      .get(url)
-      .then()
-      .statusCode(200);
+    given().param("leader", "01657nam a22002057i 4500").param("code", "000").param("headerTypeCode", "1")
+        .param("lang", "ita").headers(headers).when().get(url).then().statusCode(200);
   }
 
-
-
   @Test
   public void getFixedFieldsOfGlobe_return200Status() {
 
     String url = getURI(FIXED_FIELDS_CODE_GROUPS_URL);
     Map<String, String> headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
 
-    given()
-      .param("leader", "01657nam a22002057i 4500")
-      .param("code", "007")
-      .param("headerTypeCode", "23")
-      .param("lang", "ita")
-      .headers(headers)
-      .when()
-      .get(url)
-      .then()
-      .statusCode(200);
+    given().param("leader", "01657nam a22002057i 4500").param("code", "007").param("headerTypeCode", "23")
```

Comment:
See my earlier comments about large amounts of reformatting...

Reply:
I have changed it.